intro-to-ruby.rb

```diff
@@ -1,14 +1,14 @@
 #create a string and store it in a variable
-
+puts "My favorite food is #{favorite_food}"
```

Comment:
Remember, code is read top to bottom. Does Ruby know about `favorite_food` yet? This would throw an error.